### PR TITLE
ariane136 added for Nangate45 platform

### PR DESCRIPTION
**Pull request by vijayank88**
Signed-off-by: vijayank88 <paruthi143@gmail.com>
ariane136 test case added with macro placement using RTL-MPL2.
Design tested with 136 macros with `rtl_macro_placer`

### CODE REVIEW

File: flow/designs/nangate45/ariane136/rtlmp_config.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
+min_aspect_ratio = 0.40
```
Comment (vijayank88) [inline flow/designs/nangate45/ariane136/rtlmp_config.txt]: @ravi-varadarajan I took reference from gf12 for this setup.
But how to generate `rtlmp_config.txt` for technology specific?

File: flow/designs/nangate45/ariane136/rules-hier_rtlmp.json [hunk 2]
```diff
@@ -0,0 +1,90 @@
+{
+    "synth__design__instance__area__stdcell": {
```
Comment (maliberty) [inline flow/designs/nangate45/ariane136/rules-hier_rtlmp.json]: Why do we need this file?  That base file should suffice.

File: flow/designs/nangate45/ariane136/config_hier.mk [hunk 3]
```diff
@@ -0,0 +1,15 @@
+include $(dir $(DESIGN_CONFIG))/config.mk
```
Comment (maliberty) [inline flow/designs/nangate45/ariane136/config_hier.mk]: There is no need for a config_hier.mk.  There were two configs during migration but they are being consolidated to one.  A new test doesn't need both.

File: flow/designs/nangate45/ariane136/rules-hier_rtlmp.json (hunk 2 above)
Comment (vijayank88) [reply flow/designs/nangate45/ariane136/rules-hier_rtlmp.json]: I set `FLOW_VARIANT=hier_rtlmmp` so generated this format.
Shall I re-run with default and upload again?

File: flow/designs/nangate45/ariane136/config_hier.mk (hunk 3 above)
Comment (vijayank88) [reply flow/designs/nangate45/ariane136/config_hier.mk]: Ok. I'll merge both.

**Review by maliberty — APPROVED** [review]
Please rename constraint_hier.sdc -> constraint.sdc

**Comment by maliberty** [comment]
You should add this to jenkins/public_tests_all.Jenkinsfile so that it becomes part of the CI

**Comment by vijayank88** [comment]
@maliberty @vvbandeira 
Local flow test passed for `ariane136`, but in CI it fails during macro placement.
How to debug the issue?

**Comment by maliberty** [comment]
I just realized it also needs to be add in  jenkins/public_nightly.Jenkinsfile too.

@vvbandeira can you help with the local vs jenkins issue?

**Comment by vvbandeira** [comment]
@vijayank88 could you send me an email with the GCP machine name and path to the work folder so I can take a look?

**Comment by vijayank88** [comment]
@vvbandeira 
I've made clean build today and ran the flow again. Yes it's fail during macro placement as in CI.
I'll try to change the configuration for the fix or raise issue.

**Comment by vijayank88** [comment]
@maliberty 
I've copied constraint.sdc from private gf12 pdk. For same constraints am getting following unconstrained path
```
Floorplan check_setup
--------------------------------------------------------------------------
Warning: There are 4 unclocked register/latch pins.
Warning: There are 2391 unconstrained endpoints.
```
Is that need to be fixed? 
Can you please check private CI log for the same?

**Review by vvbandeira — APPROVED** [review]

**Comment by maliberty** [comment]
```
finish check_setup
--------------------------------------------------------------------------
Warning: There are 4 unclocked register/latch pins.
Warning: There are 2391 unconstrained endpoints.
```
should be fixed if possible

**Comment by vijayank88** [comment]
Please merge it now. We will incrementally work on it for constraints.

**Pull request merged**